src/pgrepr-consts/src/oid.rs

```diff
@@ -674,3 +674,5 @@ pub const INDEX_MZ_KAFKA_SOURCES_IND_OID: u32 = 16950;
 pub const INDEX_MZ_WEBHOOK_SOURCES_IND_OID: u32 = 16951;
 pub const TABLE_MZ_HISTORY_RETENTION_STRATEGIES_OID: u32 = 16952;
 pub const SOURCE_MZ_MATERIALIZED_VIEW_REFRESHES_OID: u32 = 16953;
+pub const VIEW_MZ_SOURCE_STATISTICS_OID: u32 = 16954;
```

Comment:
If this name is staying, it should keep its old oid. Give VIEW_MZ_SOURCE_STATISTICS_WITH_HISTORY_OID and the other new thing new oids instead.